translate tutorial/part one

SamuelAlev opened the pull request.

SamuelAlev committed:
Première ébauche de traduction

SamuelAlev committed:
Finishing translation

SamuelAlev committed:
spelling errors

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -1,190 +1,190 @@
+Dans la [**section précédente**](/tutorial/part-zero/), vous avez préparé votre environnement de développement local en installant les logiciels nécessaires et créé votre premier site Gatsby en utilisant le [**starter “hello world”**](https://github.com/gatsbyjs/gatsby-starter-hello-world). Maintenant, plongeons dans le code généré par le starter.
 
-## Using Gatsby starters
+## Utilisation des starters Gatsby
```
Not translating starter is a great idea!

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -1,190 +1,190 @@
+### ✋ Ouvrez le code
 
-In your code editor, open up the code generated for your “Hello World” site and take a look at the different directories and files contained in the ‘hello-world’ directory. It should look something like this:
+Dans votre éditeur, ouvrez le code généré pour votre site “Hello World” et prenez connaissance des différents répertoires et fichiers contenu dans le projet ‘hello-world’. Ça devrait ressembler a quelque chose comme ça :
```
```suggestion
Dans votre éditeur, ouvrez le code généré pour votre site “Hello World” et prenez connaissance des différents répertoires et fichiers contenus dans le projet ‘hello-world’. Ça devrait ressembler à quelque chose comme ça :
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -1,190 +1,190 @@
 </video>
 
-> 💡 Gatsby uses **hot reloading** to speed up your development process. Essentially, when you’re running a Gatsby development server, the Gatsby site files are being “watched” in the background — any time you save a file, your changes will be immediately reflected in the browser. You don’t need to hard refresh the page or restart the development server — your changes just appear.
+> 💡 Gatsby utilise le **rechargement immédiat** pour accélerer votre processus de développement. Lorsque vous exécutez un serveur de développement Gatsby, les fichiers du site Gatsby sont “surveillés” en arrière-plan - chaque fois que vous enregistrez un fichier, vos modifications sont immédiatement reflétées dans le navigateur. Vous n'avez pas besoin d'actualiser la page ou de redémarrer le serv ...
```
```suggestion
> 💡 Gatsby utilise le **rechargement immédiat** pour accélérer votre processus de développement. Lorsque vous exécutez un serveur de développement Gatsby, les fichiers du site Gatsby sont “surveillés” en arrière-plan - chaque fois que vous enregistrez un fichier, vos modifications sont immédiatement reflétées dans le navigateur. Vous n'avez pas besoin d'actualiser la page ou de redémarrer le serveur de développement, vos modifications apparaissent.
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -1,190 +1,190 @@
+### Attendez… De l'HTML dans notre JavaScript?
 
-_If you’re familiar with React and JSX, feel free to skip this section._ If you haven’t worked with the React framework before, you may be wondering what HTML is doing in a JavaScript function. Or why we’re importing `react` on the first line but seemingly not using it anywhere. This hybrid “HTML-in-JS” is actually a syntax extension of JavaScript, for React, called JSX. You can follow along with ...
+_Si vous êtes familier avec React et JSX, n'hésitez pas alors à passer cette section.._ Si vous n’avez jamais utilisé le framework React auparavant, vous vous demandez peut-être ce que HTML fait dans une fonction JavaScript. Ou pourquoi nous importons `react` sur la première ligne mais ne l’utilisons apparemment pas nulle part. Cet façon hybride «HTML-in-JS» est en fait une extension de syntaxe d ...
```
```suggestion
_Si vous êtes familier avec React et JSX, n'hésitez pas alors à passer cette section.._ Si vous n’avez jamais utilisé le framework React auparavant, vous vous demandez peut-être ce que HTML fait dans une fonction JavaScript. Ou pourquoi nous importons `react` sur la première ligne, mais ne l’utilisons apparemment pas nulle part. Cette façon hybride «HTML-in-JS» est en fait une extension de syntaxe de JavaScript pour React, appelée JSX. Vous pouvez suivre ce tutoriel sans expérience préalable de React, mais si vous êtes curieux, voici un bref aperçu…
```

dantereve commented on docs/tutorial/part-one/index.md:
````diff
@@ -1,190 +1,190 @@
 ```
 
-Now you can spot the use of the `'react'` import! But wait. You’re writing JSX, not pure HTML and JavaScript. How does the browser read that? The short answer: It doesn’t. Gatsby sites come with tooling already set up to convert your source code into something that browsers can interpret.
+Vous pouvez maintenant voir l’utilisation de l’importation `'react'`! Mais attendez. Vous écrivez en JSX, et non pas en HTML pur et en JavaScript. Comment le navigateur lit-il cela? La réponse courte: ce n’est pas le cas. Les sites Gatsby sont fournis avec des outils déjà configurés pour convertir votre code source en quelque chose que les navigateurs peuvent interpréter.
````
```suggestion
Vous pouvez maintenant voir l’utilisation de l’import `'react'`! Mais attendez. Vous écrivez en JSX, et non pas en HTML pur et en JavaScript. Comment le navigateur lit-il cela? La réponse courte: ce n’est pas le cas. Les sites Gatsby sont fournis avec des outils déjà configurés pour convertir votre code source en quelque chose que les navigateurs peuvent interpréter.
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -319,70 +318,69 @@ export default () => (
 </video>
 
-The Gatsby `<Link />` component is for linking between pages within your site. For external links to pages not handled by your Gatsby site, use the regular HTML `<a>` tag.
+Le component Gatsby `<Link />` permet de lier différentes pages de votre site. Pour les liens externes non géré par votre site Gatsby, utilisez la balise HTML par défaut `<a>`.
```
```suggestion
Le composant Gatsby `<Link />` permet de lier différentes pages de votre site. Pour les liens externes non gérés par votre site Gatsby, utilisez la balise HTML par défaut `<a>`.
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -319,70 +318,69 @@ export default () => (
-- Learned about components
-- Learned about Gatsby page components and sub-components
-- Learned about React “props” and reusing React components
+- Avez appris sur les générateurs de Gatsby, et comment les utiliser pour créer des nouveaux projets
```
```suggestion
- Avez découvert les starters de Gatsby, et comment les utiliser pour créer des nouveaux projets
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -319,70 +318,69 @@ export default () => (
-- Learned about Gatsby page components and sub-components
-- Learned about React “props” and reusing React components
+- Avez appris sur les générateurs de Gatsby, et comment les utiliser pour créer des nouveaux projets
+- Avez appris sur la syntaxe de JSX
```
```suggestion
- Avez découvert la syntaxe de JSX
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -319,70 +318,69 @@ export default () => (
-- Learned about React “props” and reusing React components
+- Avez appris sur les générateurs de Gatsby, et comment les utiliser pour créer des nouveaux projets
+- Avez appris sur la syntaxe de JSX
+- Avez appris sur les composants
```
```suggestion
- Avez découvert les composants
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -319,70 +318,69 @@ export default () => (
+- Avez appris sur les générateurs de Gatsby, et comment les utiliser pour créer des nouveaux projets
+- Avez appris sur la syntaxe de JSX
+- Avez appris sur les composants
+- Avez appris sur les composants et sous-composants de pages Gatsby
```
```suggestion
- Avez découvert les composants et sous-composants de pages Gatsby
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -319,70 +318,69 @@ export default () => (
+- Avez appris sur la syntaxe de JSX
+- Avez appris sur les composants
+- Avez appris sur les composants et sous-composants de pages Gatsby
+- Avez appris sur les “props” React et la réutilisation de composants React
```
```suggestion
- Avez découvert les “props” React et la réutilisation de composants React
```

dantereve commented on docs/tutorial/part-one/index.md:
```diff
@@ -319,70 +318,69 @@ export default () => (
+- Avez appris sur les “props” React et la réutilisation de composants React
 
-Now, move on to [**adding styles to your site**](/tutorial/part-two/)!
+Maintenant, continuez avec [**ajouter du styles à votre site**](/tutorial/part-two/)!
```
```suggestion
Maintenant, continuez avec [**ajouter des styles à votre site**](/tutorial/part-two/)!
```

dantereve reviewed (changes requested):
Good job 🥳 ! I've just spotted some typos and did some suggestions to fix them.

SamuelAlev committed:
fix typos

SamuelAlev commented:
Thank you for your feedback !
I fixed the typos.

dantereve committed:
delete .vscodesettings 

This branch was created before I add the .prettierrc. It will be fine with futur commits.

dantereve committed:
Apply suggestions from code review
